uk_bin_collection/uk_bin_collection/councils/BexleyCouncil.py

```
@@ -17,26 +17,32 @@ class CouncilClass(AbstractGetBinDataClass):
-        for attempt in range(3):
-            response = requests.get(page, headers=headers, timeout=30)
+        session = requests.Session()
+        session.headers.update(headers)
+
+        # Initial request triggers server-side schedule computation and sets session cookie
+        session.get(page, timeout=60)
+
+        # Poll for results — server computes async, JS client polls ?page_loading=1
+        # with x-requested-with header to get the content fragment
+        response = None
+        for attempt in range(30):
```

Ruff (0.15.12)

[warning] Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)